requirements.txt

```diff
@@ -2,11 +2,13 @@ alembic==0.9.9
 Flask==1.0
 Flask-Migrate==2.1.1
 Flask-Script==2.0.6
-Flask-SQLAlchemy==2.3.2
+Flask-SQLAlchemy==2.5.0
 Flask-WTF==0.14.3
-psycopg2
-pylint==1.8.3
+pylint
```

Comment:
Be wary of upgrading pylint without checking on it, it can lead to surprise ci failures